includes/tracks/class-wc-tracks-client.php

```diff
@@ -137,10 +137,10 @@ public static function build_timestamp() {
 	 * @return array Identity properties.
 	 */
 	public static function get_identity( $user_id ) {
-		$jetpack_lib = trailingslashit( WP_PLUGIN_DIR ) . 'jetpack/_inc/lib/tracks/client.php';
+		$jetpack_lib = '_inc/lib/tracks/client.php';
 
-		if ( class_exists( 'Jetpack' ) && file_exists( $jetpack_lib ) ) {
-			include_once $jetpack_lib;
+		if ( class_exists( 'Jetpack' ) && file_exists( JETPACK__PLUGIN_DIR . $jetpack_lib ) ) {
```

Comment:
Why even make the dependency on `class_exists( 'Jetpack' )`.

`is_defined( 'JETPACK__PLUGIN_DIR')` should be enough?

Reply:
I'd prefer to leave this exact pattern for checking if Jetpack is available in ths codebase for consistency. There used to be many file based checks before. `is_plugin_active( 'jetpack/jetpack.php' )`.

The preferred way for asserting that Jetpack is active on a site is `class_exists( 'Jetpack' )`.